content/docs/querying/basics.md

```diff
@@ -107,6 +107,30 @@ a `job` label set to `prometheus`:
 
     http_requests_total{job="prometheus"}[5m]
 
+### Offset modifier
+
+The `offset` modifier allows changing the time offset for individual
+instant and range vectors in a query.
+
+For example, the following expression returns the value of `foo` 5
+minutes in the past relative to the current query evaluation time:
+
+    foo offset 5m
```

Comment:
For the official docs, we might want to use a name other than `foo` that is more instantly recognizable as a metric name.

Like `api_http_requests_total`.

It would also be good to include a `{...}` label selector in at least one of these examples so that people see that that syntactical combination is possible: `api_http_requests_total{method="GET"} offset 5m`